go/apps/mixnet/README.md

```diff
@@ -0,0 +1,168 @@
+Mixnet over CloudProxy
+======================
+
+A mixnet router facilitates anonymous communications among a set of peers.
+Alice wants to send Msg to Bob; she encrypts, authenticates, and sends the
+message (Msg, Bob) to Molly, a mixnet router. Molly waits until she has _n-1_
+more messages from _n-1_ other peers, then transmits the messages to their
+respective destinations simultaneously, thus anonymizing Alice among a set of n
+peers. Such a service is only useful if Molly can be trusted to not divulge the
+link (Alice, Bob). This document specifies a simple mixnet built on the
+CloudProxy platform that reduces this trust to the knowledge of a public key
+and trust in the owner of that key.
+
+Security goals and threat model
+-------------------------------
+There are three principals in our protocol: the sender, recipient, and the
+policy owner. Our goal is to design a protocol that provides anonymity for
+senders without requiring the recipient to execute the protocol. The policy
+owner controls the root policy private key which is used to attest to the
```

Comment:
"which" -> "that": this is a restrictive clause, though it could use "which" if there were a preceding comma.